### PR TITLE
added unittest for concert_detail, PUT

### PR DESCRIPTION
**Pull request by berk94**
Please double-check before merging.

### CODE REVIEW

File: API Test App/ConcertAPI/concert/views.py [hunk 1]
```diff
@@ -103,4 +105,11 @@ def user_detail(request, pk):
         user.delete()
         return HttpResponse(status=204)
 
+class Test(unittest.TestCase):    
+    def test_get_a_concert(request,pk):
+        expected_result = ''
+        unittest.TestCase.assertEqual(concert_detail(request,pk), '')
```
Comment (elifguler96) [inline API Test App/ConcertAPI/concert/views.py]: Are you sure about this? 😅 You may want to delete this.

**Comment by berk94** [comment]
Done @elifguler96

File: API Test App/ConcertAPI/concert/views.py (hunk 1 above)
Comment (berk94) [inline API Test App/ConcertAPI/concert/views.py]: Yes! I was going to integrate testing into the application itself but found a better way. I forgot to delete this one 😃

**Comment by furkansenharputlu** [comment]
Great !

**Pull request merged**